### ISSUE TITLE
1.11.1 regression: enum assignment issue

### ISSUE THREAD

**Issue by strygo** [bridgedotnet/Bridge#1020]
```
    [Flags]
    public enum TestEnum : uint
    {
        /// <summary>
        /// Indicates the lack of a category.
        /// </summary>
        None = 0,

        Flag = 1,

        FlagAlias = Flag,
    }
```

### Expected behaviour
As of 1.11.0, this behavior was allowed and worked as expected.

### Actual behaviour
Compiler crash:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Error: Object reference not set to an instance of an object.    at Bridge.Translator.MemberResolver.InitResolver(SyntaxTree syntaxTree)
   at Bridge.Translator.MemberResolver.ResolveNode(AstNode node, ILog log)
   at Bridge.Translator.Inspector.GetDefaultFieldValue(AstType type, IMemberResolver resolver)
   at Bridge.Translator.FieldBlock.WriteObject(String objectName, List`1 members, String format)
   at Bridge.Translator.FieldBlock.EmitFields(TypeConfigInfo info)
   at Bridge.Translator.FieldBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.ConstructorBlock.EmitCtorForStaticClass()
   at Bridge.Translator.ConstructorBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.ClassBlock.EmitStaticBlock()
   at Bridge.Translator.ClassBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.EmitBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.Emitter.Emit()
   at Bridge.Translator.Translator.Translate()
   at Bridge.Build.GenerateScript.Execute()	ClassLibrary5	Z:\Documents\Visual Studio 2015\Projects\ClassLibrary5\ClassLibrary5\Test.cs	7	Active
```

### Steps to reproduce the behaviour
Use provided code

**Comment by DaniilVeriga** [comment]
Thank you for the report! It is going to be fixed and released with 1.11.2 soon.

**Comment by DaniilVeriga** [comment]
@strygo, 1.11.2 has been released with the fix. Thank you again for the bug report!
